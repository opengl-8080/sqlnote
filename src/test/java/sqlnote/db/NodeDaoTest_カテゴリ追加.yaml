NODE:
-
    ID: 1
    PARENT_NODE: NULL
    IS_CATEGORY: 1
    CATEGORY_ID: 1
    SQL_ID: NULL
-
    ID: 2
    PARENT_NODE: 1
    IS_CATEGORY: 1
    CATEGORY_ID: 2
    SQL_ID: NULL

CATEGORY_NODE:
-
    ID: 1
    TITLE: TEST_A
-
    ID: 2
    TITLE: TEST_B

