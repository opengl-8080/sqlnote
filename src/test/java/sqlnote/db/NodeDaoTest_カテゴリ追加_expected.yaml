NODE:
-
    ID: 1
    PARENT_NODE: NULL
    IS_CATEGORY: 1
    CATEGORY_ID: 1
    SQL_ID: NULL
-
    ID: 2
    PARENT_NODE: 1
    IS_CATEGORY: 1
    CATEGORY_ID: 2
    SQL_ID: NULL
-
    ID: 3
    PARENT_NODE: 2
    IS_CATEGORY: 1
    CATEGORY_ID: 33
    SQL_ID: NULL

CATEGORY_NODE:
-
    ID: 11
    TITLE: TEST_A
-
    ID: 22
    TITLE: TEST_B
-
    ID: 33
    TITLE: テストカテゴリ

